### DISCUSSION TITLE
God Of War 2018 Artwork

### DISCUSSION THREAD

**Discussion by M1HA15** [Atlas-OS/Atlas#428] [Community Artwork]
I'm not very good at editing pictures, but I tried. I finished God Of War recently, so I thought I'd create some artwork for fans of the series.

![gow3](https://user-images.githubusercontent.com/67576278/209226471-28ec7050-a3c6-425d-bd5c-9ffd59fbeaaa.png)
![gow4](https://user-images.githubusercontent.com/67576278/209226479-9bb72b07-78e3-4934-8f45-d410540c1bc9.png)
![gow5](https://user-images.githubusercontent.com/67576278/209226489-bbdd1f5e-0118-4db6-a032-ef85ffff48bd.png)
![gow6](https://user-images.githubusercontent.com/67576278/209226502-5f585f69-6944-432d-896b-0f7bb98dd1ce.png)
![gow7](https://user-images.githubusercontent.com/67576278/209226520-a1d7a362-7f79-43af-ba82-e8c52adbd11f.png)
![gow8](https://user-images.githubusercontent.com/67576278/209226527-51797bd6-919c-41c7-8e95-315723897917.png)
![gow9](https://user-images.githubusercontent.com/67576278/209226534-c87a578b-0b17-4207-b901-426108fd291c.png)
![gow10](https://user-images.githubusercontent.com/67576278/209226548-2ce404e3-265c-408d-8a81-826e8c8d60f5.png)
![gow1](https://user-images.githubusercontent.com/67576278/209226556-9141b4a5-6b20-4d89-ae11-4953a1eb28ec.png)
![gow2](https://user-images.githubusercontent.com/67576278/209226577-43c0aded-3686-41d7-8de9-d5194f0d572b.png)

**Comment by V1T1isclean** [comment]
Gg wtf u are so good
